### DISCUSSION TITLE
extension doesn't do anything?

### DISCUSSION THREAD

**Discussion by lukepighetti** [sweetpad-dev/sweetpad#48] [Any discussions (ideas, questions, ...)]
I'm 100% sure I have something setup wrong but I have Xcode installed, I installed SweetPad, and when I view Swift files in my ios folder in a Flutter repository i don't get any LSP, autocomplete, formatting, etc. It's just a dead text file with syntax highlighting (provided by Vscode)

Am i missing a configuration step? Does this extension expect to be used in a specific context?

Thank you!

**Comment by lukepighetti** [comment]
Did a little more digging and found the docs here: https://sweetpad.hyzyla.dev

**Reply by hyzyla** [reply]
Hi! Thanks for trying my extension. The main page in the documentation about LSP is [here](https://sweetpad.hyzyla.dev/docs/autocomplete). Basically, it heavily relies on the official [Swift extension](https://marketplace.visualstudio.com/items?itemName=sswg.swift-lang) and [xcode-build-server](https://github.com/SolaWing/xcode-build-server).

UPD: I've just read it again and realized that the problem is solved

**Reply by hyzyla** [reply]
If you have any ideas on how to improve the iOS part of the Flutter project, I'm open to suggestions ♥️

**Reply by lukepighetti** [reply]
it would be great if the lsp worked in a monorepo! flutter projects store the workplace in /ios
